README.md

```diff
@@ -34,7 +38,8 @@ For any env var that is required by the application to run and doesn't have a
 default [require-environment-variables](https://www.npmjs.com/package/require-environment-variables)
 is used to throw an error and prevent the application from starting up. Rather
 than it getting to point somewhere later in the lifecycle where it can't do
-something because there is no value for an env var it was relying on.
+something because there is no value for an env var it was relying on. 
+These env variabled are left here for reference, however the application is using `docker` to run
```

Comment:
What does this mean? The application still uses the env vars even if it is using `docker`